CRM/Banking/PluginImpl/Exporter/CSV.php

```diff
@@ -273,7 +273,15 @@ protected function apply_rule($rule, &$data_blob) {
           $data_blob[$rule->to . $key] = $value;
         }
       }
-
+    } elseif ($rule->type == 'append') {
+        // APPEND appends the string to a give value
+        if (isset($rule->seperator)){
+            // the user defined a concat string
+            $data_blob[$rule->to] = $data_blob[$rule->to].$rule->seperator.$from_value;
+        }else{
+            // default concat string is " "
+            $data_blob[$rule->to] = $data_blob[$rule->to]." ".$from_value;
+        }
```

Comment:
This can be simplified:
```suggestion
        $data_blob[$rule->to] .= $rule->seperator ?? ' ' . $from_value;
```

Reply:
Should there be a check for whether `$from_value` actually contains anything in order to avoid appending the separator only?

Reply:
I think that the original code is easier to read and I would prefer to keep it in the same way it was also implemented in the importer